app/src/main/java/org/mozilla/tv/firefox/ext/Locale.kt

```diff
@@ -17,3 +18,10 @@ fun Locale.languageAndMaybeCountryMatch(allowedLocales: Array<out Locale>?): Boo
         return@any languageMatches && countryMatches
     }
 }
+
+fun Locale.languageAndMaybeCountryMatch(allowedLocales: KillswitchLocales): Boolean {
```

Comment:
nit: I wonder if we should point the kdoc for this function to the other function.